src/spectrecoff-cli/commands/Status.fs

```diff
@@ -22,16 +22,16 @@ type StatusExample() =
             task {
                 do! Task.Delay(3000)
                 let harderThinkingSpinner =
-                    { normalThinkingSpinner with 
-                        Message = "Thinking harder..."; 
+                    { normalThinkingSpinner with
+                        Message = "Thinking harder...";
                         Look = Some { calmLook with Color = Some Color.DarkOrange } }
                 updateWithCustomSpinner harderThinkingSpinner context |> ignore
 
                 do! Task.Delay(3000)
                 let maximumThinkingSpinner =
-                    { normalThinkingSpinner with 
-                        Message = "Maximum thinking!!!"; 
-                        Look = Some { calmLook with Color = Some Color.Red }; 
+                    {
+                        Message = "Maximum thinking!!!";
```

Comment:
No need for `;` if you multi-line the record definition.

Reply:
Done